### DISCUSSION TITLE
How to contribute

### DISCUSSION THREAD

**Discussion by astrale-sharp** [SeaQL/FireDBG.for.Rust#27] [General]
Hey, Some of us (me) would love to help where possible, is that an option? :)

**Comment by tyt2y3** [comment]
Thank you for reaching out! May be we can start by testing various things, report a bug and see if we can work on a fix together?

**Reply by astrale-sharp** [reply]
Lovely!
It's written issue can only be created from discussion, So I'll start a discussion first right?
